java/cdk-environment/src/main/java/sleeper/environment/cdk/buildec2/BuildEC2Parameters.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.environment.cdk.buildec2;
+
+import sleeper.environment.cdk.config.AppContext;
+import sleeper.environment.cdk.config.AppParameters;
+import sleeper.environment.cdk.config.StringParameter;
+
+public class BuildEC2Parameters {
+
+    public static StringParameter REPOSITORY = AppParameters.BUILD_REPOSITORY;
+    public static StringParameter FORK = AppParameters.BUILD_FORK;
+    public static StringParameter BRANCH = AppParameters.BUILD_BRANCH;
+
+    private final String repository;
+    private final String fork;
+    private final String branch;
+
+    private BuildEC2Parameters(AppContext context) {
+        repository = context.get(REPOSITORY);
+        fork = context.get(FORK);
+        branch = context.get(BRANCH);
+    }
+
+    public static BuildEC2Parameters from(AppContext context) {
+        return new BuildEC2Parameters(context);
+    }
+
+    String fillUserDataTemplate(String template) {
```

Comment:
Minor: public visibility as other variables in this class are public? Plus private methods below public methods?

Reply:
I've made the static constructor package private, so now the only thing public is the build parameter constants as an alternative to referencing them in AppParameters.